include/circt/Dialect/FIRRTL/FIRRTLStructure.td

```diff
@@ -86,7 +87,6 @@ class FIRRTLModuleLike<string mnemonic, list<Trait> traits = []> :
 
 }
 
-
```

Comment:
Mega-nit: feel free to batch whitespace changes into NFC commits when appropriate.